### PR TITLE
Desktop version 

### PR DESCRIPTION
**Pull request by ayebaishmo**
In this project:
Added  media queries
Edited the CSS to fit media queries for desktop version
Added images to be used in the desktop version.
Added responsiveness to the site.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -415,15 +569,421 @@ textarea {
...
+    background-position: right bottom;
+    height: 870px;
+  }
+
+  .intro {
+    flex-direction: row-reverse;
+    justify-content: center;
+    align-items: center;
+  }
+
```
Comment (DeliceLydia) [inline style.css:635]: - [ ] I would suggest that you give the `intro` section a `width: 100%` to fill the whole container here is the image without it
wd

File: style.css [hunk 2]
```diff
@@ -415,15 +569,421 @@ textarea {
...
+  .about-me-intro .headline {
+    flex-direction: column;
+    justify-content: center;
+    align-items: flex-start;
+    margin: 141px;
```
Comment (DeliceLydia) [inline style.css:646]: - [ ] I think this margin is messing up your page try it without it and see the change.

File: style.css [hunk 3]
```diff
@@ -415,15 +569,421 @@ textarea {
...
+  }
+
+  .card-works {
+    margin: 0 auto 0 auto;
+    justify-content: flex-end;
+    align-items: baseline;
+    background-position: center;
+    background-repeat: no-repeat;
+    transition: all 0.3s ease-in;
+  }
```
Comment (DeliceLydia) [inline style.css:849]: - [ ] Please add some margin around the projects cards check how it looks now
cte

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @ayebaishmo , 

Good job so far! However, there are some issues that you still need to work on to go to the next project but you are almost there!



## Required Changes ♻️
_Check the comments under the review._


### Cheers and Happy coding!👏👏👏

Feel free to leave a question, or a comment on the PR thread if something is not clear 100%. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 3 above)
Comment (DeliceLydia) [reply style.css]: - [ ] KIndly check how the original looks like
dst

yours
destr

Comment (DeliceLydia) [reply style.css]: Please fix that and you will be good to go! and don't give up they say "Feedback is the breakfast of champions"😎

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @ayebaishmo , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._


### Keep your head up!👏👏👏

Feel free to leave a question, or a comment on the PR thread if something is not clear 100%. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ayebaishmo** [comment]
> 
> ## Hi @ayebaishmo,
> 
> Great work on making the changes requested previously 👏
> You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
> ## Required Changes ♻️
> 
> _Check the comments under the review._
> ### Keep your head up!👏👏👏
> 
> Feel free to leave a question, or a comment on the PR thread if something is not clear 100%.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Mine seems to be working fine. May I inquire which browser you are using. Cause this  is my last review. And cannot really find a solution to it.

**Review by Shaher-11 — APPROVED** [review]
Hello @ayebaishmo ,

# Congratulations! 🎉 
#### Your project is approved!  There is nothing else to say other than... it's time to merge it :muscle: :muscle: 

## Optional Suggestions 💡
_The following point is optional to implement however it's highly recommended!_:blush:

- Please stick to the given design in the about me section as shown below 

right1


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Cheers and Happy coding!👏👏👏

**Pull request merged**